docs/service_lifecycle.md

```diff
@@ -0,0 +1,62 @@
+# Service Lifecycle
+
+This defines the current service lifecycle stages, this isn't specific to the hosting platform
+and will no doubt move to somewhere more centric and referenced. However, this is a starting 
+point to clearly communication the phases of our evolution. 
+
+## Alpha
+
+Alpha is an experimental phase to test the hypothesis, by building prototypes to validate
+the direction and the intention of the service. It is there to explore ways of achieving and meeting
+the user needs in the right way. 
+
+There is more information on this [here](https://www.dta.gov.au/standard/service-design-and-delivery-process/) and
+[here](http://ausdto.github.io/service-handbook/alpha/1-introduction/1-1-what.html)
+
+
+## Beta
+
+This is the phase where software is more feature complete, it is aimed at being functional and meeting the definition
+of a Minimal Viable Product, (MVP). The service is accessible and secured and meets a lot of the production requirements and needs,
```

Comment:
....production requirements and needs. However, there will be an assessment of the service through closed user groups to evaluate that it meets the needs as expected.